bin/Data/Fonts/Anonymous Pro.xml

```diff
@@ -0,0 +1,9 @@
+<font>
+<absolute x="0" y="0" />
+<absoluteoffset x="0" y="0" />
+<scaled x="0" y="0" />
+<scaledoffset x="0" y="0" />
+<precache charset="utf8">
+<![CDATA[ !\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~]]>
+</precache>
+</font>
```

Comment:
Can you please add some indentation and newline at the end of file, in both fonts?